app/config/config.php

```diff
@@ -146,6 +146,21 @@
     'schema_filter'  => '~^(?!'.MAUTIC_TABLE_PREFIX.'messenger_messages)~',
 ];
 
+if ($container->hasParameter('mautic.db_host_ro')) {
```

Comment:
Can't get this working without the slave connection. This condition returns true even when null is set and `$container->getParameter('mautic.db_host_ro')` returns `string(41) "%env(nullable:resolve:MAUTIC_DB_HOST_RO)%"`

Reply:
@patrykgruszka thanks for reviewing this, I used the suggestion below. I can see your point